antora-playbook.yml

```diff
@@ -0,0 +1,17 @@
+site:
+  title: "Cloud Doc Test"
+  start_page: "cloud4:overview:index.adoc"
+
+content:
+  sources:
+    - url: .
+      branches: HEAD
+      start_paths: [modules/cloud4, modules/cloud]
```

Comment:
@samzhou2 Shouldn't this say modules/savanna instead of modules/cloud4?

Reply:
I changed it to savanna, but I think this file was only used for testing in the past.